### ISSUE TITLE
Define a formater to sort imports (isort-like)

### ISSUE THREAD

**Issue by jimenezrick** [scop/emacs-ruff-format#1]
It's a bit counterintuitive but `ruff` by default doesn't sort imports (as explained in https://github.com/astral-sh/ruff/issues/8367). Could we add a second formatter doing it as documented? The user would simply need to add a second hook to trigger it:

```
(reformatter-define ruff-isort
    :program ruff-format-command
    :args (list "check" "--select" "I" "--fix" "--stdin-filename" (or (buffer-file-name) input-file))
    :lighter " RuffIsort"
    :group 'ruff-format)
```

Thanks!

**Comment by Duy-X** [comment]
I agree that it would be a good idea. I actually modified my local copy with something similar:

```elisp
;;;###autoload (autoload 'ruff-fix-buffer "ruff-fix" nil t)
;;;###autoload (autoload 'ruff-fix-region "ruff-fix" nil t)
;;;###autoload (autoload 'ruff-fix-on-save-mode "ruff-fix" nil t)
(reformatter-define ruff-fix
  :program ruff-format-command
  :args (list "check" "--fix" "--exit-zero" "--stdin-filename" (or (buffer-file-name) input-file))
  :lighter " RuffFix"
  :group 'ruff-format)
```

Would be good to incorporate it into the package. I can make a PR @scop thinks this is a good idea.

**Comment by scop** [comment]
I should have added a note about this a long time ago already, but literally minutes after this package initially ended up in melpa, I discovered apheleia and have not been using this one since. I tried to get it yanked, but that did not work out.

So as it stands, this package is effectively unmaintained at the moment, and I'm not comfortable making changes to it as I'm not using it (and have no time to just test and update it for others). Sorry about the trouble. If someone wants to take it over, let's talk.

**Comment by jimenezrick** [comment]
Thanks @scop! I might still use your package as it is for a while. But I definitely will have a look at: https://github.com/radian-software/apheleia

**Comment by JoshHayes** [comment]
> So as it stands, this package is effectively unmaintained at the moment, and I'm not comfortable making changes to it as I'm not using it (and have no time to just test and update it for others). Sorry about the trouble. If someone wants to take it over, let's talk.

I have been using this package daily for a long time and already implemented a reformatter for ruff isort in my private config. I would be interested in taking over maintenance and extending the functionality to include isort. 
At some point in the future, the linting and formatting may be combined in ruff; astral-sh/ruff#8232

**Comment by scop** [comment]
Thanks @JoshHayes, and sorry about the delay, I had missed your message above.

If you're still up for it, I can initiate transfer of the repo to you -- let me know and I'll go click around.
https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository
